src/index.js

```diff
@@ -68,7 +68,14 @@ function bindEvents (options) {
     documentCapture: () => options.onDocumentCapture(getCaptures().documentCapture),
     documentBackCapture: () => options.onDocumentCapture(getCaptures().documentBackCapture),
     faceCapture: () => options.onFaceCapture(getCaptures().faceCapture),
-    complete: () => options.onComplete(getCaptures()),
+    complete: () => {
+      const captures = getCaptures();
+
+      const takenCaptures = mapValues(captures, value => !!value)
+      Tracker.sendEvent('completed flow', takenCaptures)
```

Comment:
Looks good, is it worth sending events on document and face so we know how far they got and where they dropped out?